portfolio-project-matching-app/components/FilterButtons.js

```diff
@@ -24,10 +24,14 @@ const FilterButtons = ({ category, choices, onClick }) => {
         setAvailableChoices([...availableChoices, choice])
         // pass filter choice to parent
         onClick(choice)
-        console.log(selectedChoices)
-        console.log(availableChoices)
+        // console.log('selectedChoices: ', selectedChoices)
+        // console.log('availableChoices: ', availableChoices)
     }
 
+    useEffect(() => {
+        setAvailableChoices(choices)
+    }, [choices])
+
```

Comment:
This is apparently the way to go about this rather than initializing using props.